chromium/pages/options/index.html

```diff
@@ -4,12 +4,21 @@
     <meta charset="utf-8">
     <title></title>
     <link href="style.css" rel="stylesheet">
+    <script src="../util.js" defer></script>
+    <script src="../punycode.js" defer></script>
+    <script src="ux.js" defer></script>
+    <script src="../translation.js" defer></script>
```

Comment:
`async`?

Reply:
Also move to end of `<body>`.

Reply:
I did not use `async` because scripts have dependencies on each other. `ux.js` and `translation.js` can be made into `async` because they wait for "DOMContentLoaded" anyway, but they do need `util.js` and `punycode.js` to be loaded in advance. I think I can make `translation.js` into `async`, but not the others.

Reply:
I [moved](https://github.com/EFForg/https-everywhere/pull/17890/commits/395bb05414aa5209604a7994ad0f860c27fa90de) those to the end of `<body>` and made `translation.js` `async`, but `ux.js` still needs `util.js` and `punycode.js` to run before it so I mad them `defer` to preserve order.